manifests/4.8/ptp-operator.v4.8.0.clusterserviceversion.yaml

```diff
@@ -107,6 +107,8 @@ spec:
       kind: Namespace
       metadata:
         name: openshift-ptp
+        annotations:
```

Comment:
please open PR also to add this in to 
1. https://github.com/openshift-kni/cnf-features-deploy/blob/master/feature-configs/deploy/ptp/01_namespace.yaml
2. https://github.com/openshift/openshift-docs/blob/master/modules/nw-ptp-installing-operator.adoc